server/controllers/profile.controller.js

```diff
@@ -0,0 +1,11 @@
+const User = require('../models/user');
+ 
+async function findByUserName(user) {
+  return await User.findOne({email : user});
+}
+
+async function updateProfile(user) {
+    return await User.updateOne({ _id: user._id }, user);
```

Comment:
where is caloriesNeeds calculate?

Reply:
done https://github.com/vnts7/mwa-prj/pull/10/commits/e603276aa1a232976b2e4fd50d1075283c85a49c